sql/tables.sql

```diff
@@ -0,0 +1,113 @@
+     `staff_role_description` VARCHAR(12) CHARACTER SET utf16 NOT NULL,
+     `wsr_access_level`       DECIMAL(10, 0) DEFAULT NULL,
+     `wsr_group`              VARCHAR(30) CHARACTER SET utf16 DEFAULT NULL,
+     `ns_week_hrs`            DECIMAL(10, 0) NOT NULL,
+     `ns_year_hrs`            DECIMAL(10, 0) NOT NULL,
+     `postcode`               VARCHAR(80) CHARACTER SET utf16 NOT NULL,
+     `lat`                    FLOAT NOT NULL,
+     `lng`                    FLOAT NOT NULL,
+     `home_postcode`          VARCHAR(80) CHARACTER SET utf16 NOT NULL,
+     `home_lat`               FLOAT NOT NULL,
+     `home_lng`               FLOAT NOT NULL,
+     `deploy_postcode`        VARCHAR(80) CHARACTER SET utf16 NOT NULL,
+     `deploy_lat`             FLOAT NOT NULL,
+     `deploy_lng`             FLOAT NOT NULL,
+     `deploy_flag`            SMALLINT(6) NOT NULL,
+     `ns_business_unit`       VARCHAR(60) CHARACTER SET utf16 DEFAULT NULL,
+     `ns_ngt_start`           DATETIME NOT NULL,
+     `active_flag`            SMALLINT(6) NOT NULL,
+     `nsf_view_flag`          VARCHAR(3) CHARACTER SET utf16 NOT NULL,
+     `deleted`                SMALLINT(6) NOT NULL,
+     PRIMARY KEY (`id`),
+     KEY nsf_staff_ind1 (`staff_id`)
+  )
+ENGINE = InnoDB;
+
+CREATE TABLE IF NOT EXISTS `ngt_site_events`
+  (
+     `id`                DECIMAL(10, 0) NOT NULL,
+     `site_event_id`     DECIMAL(10, 0) NOT NULL,
+     `site_id`           DECIMAL(10, 0) NOT NULL,
+     `cms_org_id`        VARCHAR(9) CHARACTER SET utf16 NOT NULL,
+     `site_type`         VARCHAR(6) CHARACTER SET utf16 NOT NULL,
+     `skills_group`      VARCHAR(12) CHARACTER SET utf16 DEFAULT NULL,
+     `event_date`        DATETIME NOT NULL,
+     `event_start`       VARCHAR(10) NOT NULL,
+     `event_end`         VARCHAR(10) NOT NULL,
+     `event_duration`    VARCHAR(12) CHARACTER SET utf16 NOT NULL,
+     `durationm`         DECIMAL(10, 0) NOT NULL,
+     `status`            VARCHAR(30) CHARACTER SET utf16 NOT NULL,
+     `workflow_stage`    DECIMAL(2, 1) NOT NULL,
+     `p_booked`          SMALLINT(6) NOT NULL,
+     `confirmed_flag`    SMALLINT(6) NOT NULL,
+     `type_booking`      VARCHAR(9) CHARACTER SET utf16 NOT NULL,
+     `scheduled`         SMALLINT(6) NOT NULL,
+     `staff_id`          DECIMAL(10, 0) NOT NULL,
+     `cnx_email`         SMALLINT(6) NOT NULL,
+     `nse_order_num`     VARCHAR(15) CHARACTER SET utf16 DEFAULT NULL,
+     `nse_split_event`   SMALLINT(6) NOT NULL,
+     `nse_multi`         SMALLINT(6) NOT NULL,
+     `on_call_completed` SMALLINT(6) NOT NULL,
+     `deleted`           SMALLINT(6) NOT NULL,
+     `created`           DATETIME DEFAULT NULL,
+     `created_by`        DECIMAL(10, 0) NOT NULL,
+     PRIMARY KEY (`id`),
+     KEY nse_site_event_ind1 (`site_event_id`),
+     KEY nse_site_ind2 (`site_id`),
+     KEY nse_staff_ind3 (`staff_id`),
+     KEY nse_event_date_ind4 (`event_date`)
+  )
+  ENGINE = InnoDB;
+  
```

Comment:
Throughout all table definitions table names are lower case. CVS Staging database runs with `lower_case_table_names` set to `0` that means name comparisons are case-sensitive. DMS creates tables upper-case in preprod/prod - I assume tables are referred in a similar manner in Lambdas (VTM/WMS).

Reply:
The table and field names are also all capitalised in dev/int (see queries to generate test data for examples). Great spot Chris.

Reply:
Will update so that everything is uppercase.  

The reversed table creation scripts I got from prod had everything lowercase, but that's probably the IDE doing it.